GeoTIFF_Standard/standard/abstract_tests/TIFF_Tests/TEST_TIFF_Tags.adoc

```diff
@@ -73,6 +73,11 @@ Process each IFD using the following pseudocode:
             Validate that Bytes 2-3 = 12 (Double)
             Set DoubleValues to the value in Bytes 8-11
             }
+        IF value = 34738 THEN
+            {
+            Validate that Bytes 2-3 = 4 (Long or ULong)
```

Comment:
For consistency with the TIFF spec, we should only mention Long, which is a unsigned 32-bit integer